lib/tasks/close_rejected_offender_sars_task.rake

```diff
@@ -0,0 +1,9 @@
+namespace :close_rejected_offender_sars do
+  desc "Close rejected offender sars that were received over the deadline"
+  task update_statuses: :environment do
+    CaseSAROffender.scope: "late".each do |rejected_offender_sar|
```

Comment:
I'm not sure what `CaseSAROffender` is.
You would normally call a scope directly e.g. `Object.scope_name`

Reply:
Shouldn't this only apply to rejected cases?

Reply:
Amended the criteria using rejections only and with a scope of late